src/components/MenuLeft.vue

```diff
@@ -1,3 +1,34 @@
+<script setup lang="ts">
+import { useTreeNodes } from "@/store/treeNodes";
+
+const nodes = useTreeNodes().$state.items;
+</script>
+
 <template>
-  <nav class="w-3">Menu</nav>
+  <nav
+    class="left-menu bg-white"
+    :class="nodes.length > 14 ? 'overflow-y-scroll' : 'overflow-hidden'"
+  >
+    <PanelMenu class="mt-3" :model="nodes" />
+  </nav>
 </template>
+
+<style>
+.p-panelmenu .p-panelmenu-header .p-panelmenu-header-content {
```

Comment:
can you use existing classes instead of overriding them in CSS?

Reply:
Ur right, I can, fixed